text/3579-pki.md

```diff
@@ -0,0 +1,262 @@
+
+Authentication to the CloudHSM instance shall be limited to only a [Signing Console][signing-console] at the network layer; at the application layer, we shall create IAM roles and accounts for all members of the quorum. These accounts will required two factor authentication and have restrictive password, timeout, and security policies applied. These roles shall only be used for Quorum signing events. There will be documentation for the quorum members on best practices for handling their credentials.
+
+These individuals should be available in the event quorum is needed for root key operations. These roles can and should be reappointed as needed, but to simplify logistics, these roles should not require rotation more often than 2-3 years. (Operations requiring quorum are expected to be rare, and an annual rotation would make rotation of the quorum group the vast majority of quorum operations.)
+
+##### When the Quorum will be needed
+[when-the-quorum-will-be-needed]: #when-the-quorum-will-be-needed
+
+- Changes to quorum membership or thresholds shall require a quorum (enforced by the HSM authentication)
+- Replacement of the Root Key in the event of expiration or compromise
+- Creation, signing, and issuance of a new Delegate Key
+- Creation, signing, and issuance of a replacement Delegate Key in the event of expiration or compromise
+
+##### Quorum Threat Mitigations
+
+- To reduce the feasibility of coercion or collusion, we recommend that no more than 3 (M-2) of the 9 people be affiliated with any one company or institution, or be residents or citizens of the same country. (If in the future the quorum threshold is changed, this limit should be changed accordingly to remain M-2).
+- If in the future a desire arises to modify the quorum, we strongly recommend growing the size of membership rather than reducing the quorum threshold. The quorum exists to mitigate against bad actors, and is not implemented to prove a majority.
+- To reduce the feasibility of coercion or collusion, we recommend that members of the quorum be selected from within the Rust project, community, Foundation staff and/or Foundation Board. This allows us to spread trust and responsibility among socially and fiduciarily responsible parties.
+- To prevent the destruction of the quorum, we recommend that no real world event occur in which 5 or more members of the quorum are present; we recommend at least 5 (M-2) members of the quorum remain at geographically diverse locations. (If in the future the quorum threshold is changed, this limit should be changed accordingly to remain M-2).
+- Hardware keys shall be utilized for two factor authentication for Quorum IAM accounts; phone-based or weaker methods of 2FA authentication cannot be used
+
+### Delegate Keys
```

Comment:
What mechanism will be used to prevent delegate keys to be used for a different purpose than intended? For example the [crates.io](http://crates.io/) key being used for signing. Will it use the subject field, a dns constraint or a custom constraint? Or some entirely different mechanism?

Reply:
We've elaborated on this now.